config_test.go

```diff
@@ -68,6 +84,52 @@ emptydirs:
 	assert.Nil(t, testWrite(t, deb))
 }
 
+func TestExampleConfig2(t *testing.T) {
```

Comment:
Maybe name it TestExampleConfigWithControlExtraContent? Config2 is not saying very much.